packages/playground/src/explorer/utils/helpers.ts

```diff
@@ -0,0 +1,35 @@
+import type { NodesQuery } from "@threefold/gridproxy_client";
+
+import { gridProxyClient } from "@/clients";
+
+import type { MixedFilter } from "./types";
+
+export function requestNodes(options: Partial<NodesQuery>, loadFarm?: boolean) {
+  return gridProxyClient.nodes.list(options, { loadFarm: loadFarm });
+}
+
+export const multiplyResource1024 = (resource: number | undefined): number => {
+  return resource ? resource * 1024 * 1024 * 1024 : 0;
+};
```

Comment:
The naming is really weird maybe update it to `toBytes` or whatever is converts to